build/buildConfig.js

```diff
@@ -302,44 +304,9 @@ module.exports = (...args) => mapArgumentsToObject(args, ({
             loader: 'html-loader'
         }] : [])
     },
-    devServer: devServer || {
-        publicPath: "/dist/",
-        proxy: proxy || {
-            '/rest': {
-                target: "https://dev-mapstore.geosolutionsgroup.com/mapstore",
-                secure: false,
-                headers: {
-                    host: "dev-mapstore.geosolutionsgroup.com"
-                }
-            },
-            '/pdf': {
-                target: "https://dev-mapstore.geosolutionsgroup.com/mapstore",
-                secure: false,
-                headers: {
-                    host: "dev-mapstore.geosolutionsgroup.com"
-                }
-            },
-            '/mapstore/pdf': {
-                target: "https://dev-mapstore.geosolutionsgroup.com",
-                secure: false,
-                headers: {
-                    host: "dev-mapstore.geosolutionsgroup.com"
-                }
-            },
-            '/proxy': {
-                target: "https://dev-mapstore.geosolutionsgroup.com/mapstore",
-                secure: false,
-                headers: {
-                    host: "dev-mapstore.geosolutionsgroup.com"
-                }
-            },
-            '/docs': {
-                target: "http://localhost:8081",
-                pathRewrite: {'/docs': '/mapstore/docs'}
-            }
-        }
-    },
-
-    devtool: !prod ? 'eval' : undefined
+    devServer: proxy
+        ? { publicPath: "/dist/", proxy } // backward compatibility
+        : devServer,
+    devtool: !prod ? 'eval' : devtool || undefined
```

Comment:
I think also this statement is changing the previous behaviour, devServer had priority before

Reply:
yes, I though it was an acceptable change. If it is not, I'm going to add logic to support it ?